core/src/main/java/feast/core/dao/FeatureSetRepository.java

```diff
@@ -36,6 +36,12 @@ FeatureSet findFirstFeatureSetByNameLikeAndProject_NameOrderByVersionDesc(
   // find all feature sets and order by name and version
   List<FeatureSet> findAllByOrderByNameAscVersionAsc();
 
+  // find all feature sets by name and project name
+  List<FeatureSet> findAllByNameAndProject_Name(String name, String projectName);
```

Comment:
I dont think you need these methods, might be better to inject the `SpecService` and use `ListFeatureSets`

Reply:
I refactored `JobService` to use `SpecService` to get its featuresets. The new methods have been removed.